src/PrizePool.sol

```diff
@@ -391,20 +402,22 @@ contract PrizePool is Manageable, Multicall {
         
         (UD60x18 deltaExchangeRate, uint256 remainder) = _computeDrawDeltaExchangeRate(nextNumberOfTiers);
         prizeTokenPerShare = prizeTokenPerShare.add(deltaExchangeRate);
+        // console2.log("prizeTokenPerShare", prizeTokenPerShare.unwrap());
 
         uint256 _additionalReserve = fromUD60x18(deltaExchangeRate.mul(toUD60x18(reserveShares)));
         _reserve += _additionalReserve + reclaimedLiquidity + remainder;
 
         return lastCompletedDrawId;
     }
 
-    /// @notice Reclaims liquidity from a tier
+    /// @notice Reclaims liquidity from tiers, starting at the highest tier
     /// @param _count The number of tiers to reclaim liquidity for
     /// @return The total reclaimed liquidity
     function _reclaimTierLiquidity(uint8 _count) internal view returns (uint256) {
         uint256 reclaimedLiquidity;
-        for (uint8 i = 0; i < _count; i++) {
+        for (uint8 i = numberOfTiers - _count; i < numberOfTiers; i++) {
```

Comment:
`numberOfTiers` should be passed to `_reclaimTierLiquidity` since we sload it once in `completeAndStartNextDraw`:
https://github.com/pooltogether/v5-prize-pool/blob/01a28037121ed67496fc547567e45ebeb91b1ab2/src/PrizePool.sol#L367

Also, `numTiers` should be used instead of `numberOfTiers` to initialize `nextNumberOfTiers` to avoid one sload:
https://github.com/pooltogether/v5-prize-pool/blob/01a28037121ed67496fc547567e45ebeb91b1ab2/src/PrizePool.sol#L368

`numberOfTiers - _count` is equivalent to `numberOfTiers - (numberOfTiers - nextNumberOfTiers)` which is equivalent to `nextNumberOfTiers`.
So we can refactor it this way:
```suggestion
        for (uint8 i = nextNumberOfTiers; i < numberOfTiers; i++) {
```